daniellamarr/warren-web
# Implement scroll functionality to navigation

#### What does this PR do?
- This PR adds a scroll functionality to the navigation bar
#### Description of Task to be completed?
- When a link is being clicked on the navigation bar, it should scroll to the expected section
- When scrolling down the website, the navbar should become fixed so as to allow the navigation flow with the webpage
#### How should this be manually tested?
- Pull this branch `implement-scroll-functionality-to-nav`
- Navigate to the root `index.html` file
- Scroll down the webpage and watch the navbar stick to the top of the webpage
- Click on a link on the navbar to get to the expected section
#### Screenshots (if appropriate)
<img width="1439" alt="Screen Shot 2019-04-28 at 11 40 59 PM" src="https://user-images.githubusercontent.com/42177767/56870961-3489a380-6a0f-11e9-9982-66dd645f6670.png">
